src/components/Calculator.js

```diff
@@ -0,0 +1,36 @@
+function Calculator() {
+  return (
+    <section id="calculator">
+      <div className="operators">
+        <Display />
+        <span className="grey">AC</span>
+        <span className="grey">+/-</span>
+        <span className="grey">%</span>
+        <span className="orange">/</span>
+        <span className="grey">7</span>
+        <span className="grey">8</span>
+        <span className="grey">9</span>
+        <span className="orange">x</span>
+        <span className="grey">4</span>
+        <span className="grey">5</span>
+        <span className="grey">6</span>
+        <span className="orange">-</span>
+        <span className="grey">1</span>
+        <span className="grey">2</span>
+        <span className="grey">3</span>
+        <span className="orange">+</span>
+        <span className="grey zero">0</span>
+        <span className="grey">.</span>
+        <span className="orange">=</span>
+      </div>
+    </section>
+  );
+}
+
+function Display() {
+  return (
+    <span className="display">0</span>
+  );
+}
```

Comment:
[OPTIONAL] I think you should move this child component to an external file. This helps with the separation of concerns.

Reply:
Thank you, I will make changes as suggested.